src/main/java/com/epam/test_generator/services/JWTTokenService.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.test_generator.services;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.epam.test_generator.entities.User;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+import java.io.UnsupportedEncodingException;
+
+@Service
+public class JWTTokenService {
```

Comment:
Are the any cases where we should create a builder but don't use `signJWTToken`. If no I think we could have only 1 public method here like `createJwtTokenFor(User user,...)` and make other methods `private`. @hdghg What do you think about it?